pandas/tests/dtypes/test_inference.py

```diff
@@ -607,6 +635,42 @@ def test_maybe_convert_objects_nullable_integer(self, exp):
 
         tm.assert_extension_array_equal(result, exp)
 
+    @pytest.mark.parametrize(
```

Comment:
can you integrate with test right above

Reply:
If we are going to go with convert_to_nullable, no.